Failling test on metricbeat/module/kubernetes

Original job: https://beats-ci.elastic.co/job/elastic+beats+pull-request+multijob-darwin/387/beat=metricbeat,label=macosx/console

```
21:40:51 command [go test -race -cover -coverprofile /var/folders/j9/49dsjg8n2gz4zn3tqk1pcsg00000km/T/gotestcover-251885035 github.com/elastic/beats/metricbeat/module/kubernetes/util]: exit status 1
21:40:51 --- FAIL: TestTimeout (0.09s)
21:40:51 	<autogenerated>:1: 
21:40:51                           
21:40:51 	Error Trace:	metrics_cache_test.go:41
21:40:51 		
21:40:51 	Error:      	Not equal: 
21:40:51 		
21:40:51 	            	expected: 0
21:40:51 		
21:40:51 	            	actual  : 3.14
21:40:51 		
21:40:51 	Test:       	TestTimeout
21:40:51 FAIL
21:40:51 coverage: 87.1% of statements
21:40:51 FAIL	github.com/elastic/beats/metricbeat/module/kubernetes/util	0.129s
````

--- a/metricbeat/module/kubernetes/util/metrics_cache_test.go
+++ b/metricbeat/module/kubernetes/util/metrics_cache_test.go
@@ -31,7 +31,7 @@ func TestGetWithDefault(t *testing.T) {
 }
 
 func TestTimeout(t *testing.T) {
-	test := newValueMap(20 * time.Millisecond)
+	test := newValueMap(10 * time.Millisecond)
 
 	test.Set("foo", 3.14)
 	assert.Equal(t, 3.14, test.Get("foo"))

